package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "star-rating",
```

Comment:
Missing the e2e testing for this widget.

Reply:
Was Done